### DISCUSSION TITLE
Where to get chromedriver on macOS?

### DISCUSSION THREAD

**Discussion by ricardomatias** [Eloston/ungoogled-chromium#2866] [Q&A] [answered]
I installed ungoogled-chromium via homebrew and the cask eloston-chromium, but I can't find the chromedriver. Where is it located? Or should I install from a different source to have it?

**Comment by Qubik65536** [comment, marked as answer]
I found [this](https://googlechromelabs.github.io/chrome-for-testing/), is this what you're looking for?

**Reply by ricardomatias** [reply]
Yes, but I hoped there was a release channel for it that is bundled with ungoogled-chromium. Otherwise keeping matching versions becomes a manual job.

**Reply by Qubik65536** [reply]
Is it only unbundled with the macOS version, or is it not there for any of them? I am not very familiar with it and might have to investigate it a bit more later when I got more time.

**Reply by networkException** [reply]
As far as I know usually only the major version is relevant as major releases are the ones likely to include breaking changes to the IPC
